### DISCUSSION TITLE
srsRAN + bladeRF + T100 Amplifter

### DISCUSSION THREAD

**Discussion by buchhardt** [srsran/srsRAN_Project#143] [General Help] [answered]
Hello, 
I'm running srsenb  with bladeRF SDR  and it works fine. I can transmit video and receive it. 
Now I bought a small amplifter from nuand T100  
Amplifter is working fine  I can see it (led is blue), and i can switch it on and off with bladeRF-cli 

As soon as I start srsenb  LED switches of and stays off. 
Did anyone work with that amplifer  ? How can I support that amlifter and avoid to be reseted due start of the node ? (Or stay enabled during starting srsenb ? )

Or can anyone recommend an other amplifter that works with srsenb and bladerRF ? 

Gerd

**Comment by andrepuschmann** [comment, marked as answer]
Hey - I believe you're working with srsENB in the 4G repo? Perhaps better to close the issue here and reopen it there. In any case, I guess the solution is just to add the API to control the bias T from the RF driver you're using, either the Soapy or the native one. You didn't specify the one you're using.
